clang/runtime/dpct-rt/include/dpct/bindless_images.hpp

```diff
@@ -1352,6 +1425,35 @@ static inline void dpct_memcpy(image_mem_wrapper *dest, size_t w_offset_dest,
   sycl::free(temp, q);
 }
 
+// A wrapper for sycl write_image function for the byte addressing image.
```

Comment:
Seems the write_image_by_byte is not required.  let's remove it target to low learning cost of dpct helper function for end user.

Reply:
Done